opm/autodiff/StandardWellsDense.hpp

```diff
@@ -69,7 +72,9 @@ enum WellVariablePositions {
         template<typename FluidSystem, typename BlackoilIndices>
         class StandardWellsDense {
         public:
-
+            // --------- Ebos property system stuff -------
+            typedef typename TTAG(EclFlowProblem) TypeTag;
+            typedef typename GET_PROP_TYPE(TypeTag, ElementContext)    ElementContext;
```

Comment:
actually, I think that the difference between

```c++
using Foo = Bar;
```

and

```c++
typedef Bar Foo;
```

is negligible. since the former does not work with older compilers, I've stuck to the latter so far, but I don't really mind the `using` syntax.

>  Also, why not give the TypeTag a more meaningful name in the context it is used, such as Problem here for example?

the type tag already is called `EclFlowProblem`? IMO, renaming the typedef makes visual pattern matching between classes more difficult and renaming the `TypeTag` to `Problem` would confuse people like me into thinking that is the name of the problem class (in this case `EclProblem`, there is a property called `Problem`)?